backend-rust/migrations/0001_initialize.up.sql

```diff
@@ -206,17 +205,27 @@ CREATE TABLE accounts(
         -- Starting at 1 to count the transaction that made the account.
         DEFAULT 1,
     -- The total delegated stake of this account in micro CCD.
-    -- TODO: Actually populate this in the indexer.
     delegated_stake
         BIGINT
         NOT NULL
-        DEFAULT 0
+        DEFAULT 0,
+    -- Whether we are re-staking earnings. Null means we are not using delegation.
+    delegated_restake_earnings
+        BOOLEAN
+        NULL,
+    -- Target id of the baker When this is null it means that we are using passive delegation.
+    delegated_target_baker_id
+        BIGINT
+        NULL
 );
 
 -- These are important for the sorting options on the accounts query.
 CREATE INDEX accounts_amount_idx ON accounts (amount);
 CREATE INDEX accounts_delegated_stake_idx ON accounts (delegated_stake);
 CREATE INDEX accounts_num_txs_idx ON accounts (num_txs);
+CREATE INDEX accounts_delegated_target_baker_id_index
```

Comment:
Please add a comment to when this index is interesting, these details can be hard to derive later

Reply:
I removed this index. I introduced it because it was already present.

Reply:
Removing it